packages/atlas-experiment-table/src/TableManager.js

```diff
@@ -132,25 +149,32 @@ export default class TableManager extends React.Component {
   }
 
   // Filters are pairs of key-value that are searched for in the dataRows prop
-  updateFilters(dataKey, value) {
-    this.setState({
-      filters:
-        _.chain(this.state.filters)
-          .omit(dataKey)  // Returns new object, this.state.filters isn’t mutated, can _.cloneDeep if needed though
-          .assign(
-            // An empty string means the filter has been deleted from the table header or All has been selected in a
-            // dropdown, so we add no keys to the filters object
-            value.trim() === `` ? {} : { [dataKey]: value })
-          .value(),
-      currentPage: 1
-    })
+  updateFilters(dataKey, value, debounce = true) {
```

Comment:
If `debounce` is always true, do we still need this variable?

Reply:
It’s false when updating the dropdowns (have a look at how the `dropdownOnChange` prop is used in `TablePreamble`). In that case we don’t want to delaythe filtering.